tests/datasets_tests/molnet_tests/test_molnet.py

```diff
@@ -0,0 +1,146 @@
+import os
+
+import numpy as np
+import pytest
+
+from chainer_chemistry.dataset.preprocessors.atomic_number_preprocessor import AtomicNumberPreprocessor  # NOQA
+from chainer_chemistry.datasets import molnet
+
```

Comment:
Currently, only bbbp and clearance dataset test are supported?

Reply:
When I implemented it, I feel testing one regression and another classification dataset are enough. But if we would like to support dataset, we need to test all dataset.(other PR is OK?)

Reply:
other pr is ok.